### ISSUE TITLE
[Devel] AttributeError: 'FilebasedUserManager' object has no attribute 'register_callback'

### ISSUE THREAD

**Issue by koenkooi** [foosel/OctoPrint#2944]
#### What were you doing?

I am trying to update my octoprint instance running on 'devel' now that that branch has seen some activity. 

#### What did you expect to happen?

A working instance.

#### What happened instead?

Spinning dots in the browser, errors in the log:
```
Dec 07 19:45:07 raspberrypi2 octoprint[147]: 2018-12-07 19:45:06,574 - tornado.general - ERROR - Cannot send error response after headers written
Dec 07 19:45:07 raspberrypi2 octoprint[147]: 2018-12-07 19:45:06,600 - tornado.application - ERROR - Uncaught exception POST /sockjs/858/ckwbrht4/xhr_streaming?t=1544208306593 (172.20.0.13)
Dec 07 19:45:07 raspberrypi2 octoprint[147]: HTTPServerRequest(protocol='http', host='raspberrypi2.local', method='POST', uri='/sockjs/858/ckwbrht4/xhr_streaming?t=1544208306593', version='HTTP/1.1', remote_ip='172.20.0.13', headers={'Origin': 'http://raspberrypi2.local', 'Content-Length': '0', 'Accept-Language': 'en-US,en;q=0.9,nl;q=0.8', 'Accept-Encoding': 'gzip, deflate', 'X-Forwarded-For': '172.20.0.13', 'X-Scheme': 'http', 'Host': 'raspberrypi2.local', 'Accept': '*/*', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Safari/537.36', 'Dnt': '1', 'Connection': 'upgrade', 'Cookie': 'JSESSIONID=dummy; session_P80=.eJxNj8FqwzAQRH-l7DmURokvhh5q5EANu6VGjlhdTJu4sSUrFMeGSiH_XpVS6GFY5s0yMFdoP6bu0kM-T0u3gnY4Qn6Fu3fIgTyNqHc9qyawqq3RnBlZefa4JnnYkibPsfasG2HU3pN1GcrasTcWpdugKB_QNxnHKvndiOopo8RYVKORzdooF4x8jiTJUSy_jE5d6hRfdCk4KTFBthjI8ob1Pt1-YFU41K8BU4K28EYd0h8-wm0Fw7E7z8Mc7t-WuW_n8NlBfl7G8V_ys-4XLZduav_s7Ruam1if.DuxRLQ.phAfp2rfoWk3Golf8Ll00P29eh0', 'X-Real-Ip': '172.20.0.13'})
Dec 07 19:45:07 raspberrypi2 octoprint[147]: Traceback (most recent call last):
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/tornado/web.py", line 1468, in _stack_context_handle_exception
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     raise_exc_info((type, value, traceback))
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/tornado/web.py", line 1670, in wrapper
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     result = method(self, *args, **kwargs)
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/octoprint/vendor/sockjs/tornado/transports/xhrstreaming.py", line 29, in post
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     if not self._attach_session(session_id, False):
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/octoprint/vendor/sockjs/tornado/transports/pollingbase.py", line 28, in _attach_session
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     session = self.server.create_session(session_id)
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/octoprint/vendor/sockjs/tornado/router.py", line 163, in create_session
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     self.settings.get('disconnect_delay')
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/octoprint/server/util/sockjs.py", line 36, in __init__
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     octoprint.vendor.sockjs.tornado.session.Session.__init__(self, conn, server, session_id, expiry=expiry)
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/octoprint/vendor/sockjs/tornado/session.py", line 226, in __init__
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     BaseSession.__init__(self, conn, server)
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/octoprint/vendor/sockjs/tornado/session.py", line 86, in __init__
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     self.conn = conn(self)
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/octoprint/server/__init__.py", line 866, in _create_socket_connection
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     eventManager, pluginManager, session)
Dec 07 19:45:07 raspberrypi2 octoprint[147]:   File "/usr/lib/python2.7/site-packages/octoprint/server/util/sockjs.py", line 101, in __init__
Dec 07 19:45:07 raspberrypi2 octoprint[147]:     self._userManager.register_callback(self._user_manager_callback)
Dec 07 19:45:07 raspberrypi2 octoprint[147]: AttributeError: 'FilebasedUserManager' object has no attribute 'register_callback'
Dec 07 19:45:07 raspberrypi2 octoprint[147]: 2018-12-07 19:45:06,603 - tornado.general - ERROR - Cannot send error response after headers written
```

#### Did the same happen when running OctoPrint in safe mode?

GUI doesn't come up, so I can't enable safe mode.

#### Version of OctoPrint

Devel, revision 3a1611e5c4d66442ba36275545ac5563962cb500

#### Operating System running OctoPrint

Raspberrypi 2B

#### Printer model & used firmware incl. version

Prusa i3 MK3

#### Browser and version of browser, operating system running browser

Chrome 70.0.3538.110 on OSX

#### Link to octoprint.log

https://gist.github.com/koenkooi/b02974d685cdb24697859c156f51247f

#### Link to contents of Javascript console in the browser

N/A

#### Screenshot(s)/video(s) showing the problem:

N/A

I have read the FAQ.

**Comment by koenkooi** [comment]
Just tested with 17ad63dabf90da8beb2b01f7fdf7c93681f54c81, issue is stil present.

**Comment by foosel** [comment]
Should be fixed in e14cb44

**Comment by koenkooi** [comment]
Fix confirmed!
```
Dec 13 09:31:12 raspberrypi2 octoprint[11267]: 2018-12-13 09:31:12,782 - octoprint.startup - INFO - ******************************************************************************
Dec 13 09:31:12 raspberrypi2 octoprint[11267]: 2018-12-13 09:31:12,786 - octoprint.startup - INFO - Starting OctoPrint 1.4.0.dev1705+ge14cb44.dirty
Dec 13 09:31:12 raspberrypi2 octoprint[11267]: 2018-12-13 09:31:12,787 - octoprint.startup - INFO - ******************************************************************************
```
